perceval/backends/core/confluence.py

```diff
@@ -56,11 +56,12 @@ class Confluence(Backend):
 
     CATEGORIES = [CATEGORY_HISTORICAL_CONTENT]
 
-    def __init__(self, url, tag=None, archive=None):
+    def __init__(self, url, add_ancestors, tag=None, archive=None):
         origin = url
 
         super().__init__(origin, tag=tag, archive=archive)
         self.url = url
+        self.add_ancestors = add_ancestors
```

Comment:
The backend params generally include global settings (url, api token, username, etc.), specific things should go to the fetch params (e.g., `latest_items` in the git backend https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/git.py#L78)